itest/assets_test.go

```diff
@@ -1198,7 +1216,15 @@ func assertInvoiceHtlcAssets(t *testing.T, node *HarnessNode,
 
 	t.Logf("Asset invoice: %v", toProtoJSON(t, invoice))
 
-	targetID := hex.EncodeToString(assetID)
+	var targetID string
+	switch {
+	case len(assetID) > 0:
+		targetID = hex.EncodeToString(assetID)
+
+	case len(groupID) > 0:
+		groupHash := sha256.Sum256(groupID)
+		targetID = hex.EncodeToString(groupHash[:])
```

Comment:
Hmm, do we need to modify what we return as `JsonHtlc` to start to include a distinct group key field? IIRC, it's all TLV within the db/lnd, so shouldn't be a big lift there, and should be backwards compat.

Reply:
Hmm, good idea. We can definitely add that to make certain things easier. Even if it's just informative to the user and useful in itests (but not explicitly expected in the RFQ logic).